types/types.go

```diff
@@ -58,7 +58,7 @@ type Image interface {
 	IntendedDockerReference() string
 	// Manifest is like ImageSource.GetManifest, but the result is cached; it is OK to call this however often you need.
 	// NOTE: It is essential for signature verification that Manifest returns the manifest from which LayerDigests is computed.
-	Manifest() ([]byte, error)
+	Manifest() ([]byte, string, error)
```

Comment:
> At the moment there may be no difference because all our manifest types are self-describing. Is that likely to remain true?

it is to me

I think adding an empty string case makes the whole error better :)